files/imv-folder

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+image="$1"
+exec imv "$(dirname "$image")"/* -n "$image"
```

Comment:
Doesn't this include directories and cause a bit of confusion? I mean if I have:
```
- folder1
|---image1.jpg
|---folder2
          |---image2.jpg
```

This would also show image2.jpg on `next`, no?

Reply:
Ohh, I noticed some weird behaviour lately, that's it! That's correct, I can hack something around this in bash, but it might be slow on large folders.

Reply:
Not sure what's your (or project's) stance with external dependencies, but I think a quick `find` command with  `-maxdepth 1 -type f` would be sufficient to exclude folders and list only files from the specified folder :crossed_fingers:

Reply:
Yeah, I sticked to bash and implemented this. Will push it now, rebasing to the latest master.